data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -230,6 +231,36 @@ then
     record_service_stage_success
 fi
 
+if [ ! -f cloud-controller-manager-bootstrap.done ]
+then
+    record_service_stage_start "cluster-cloud-controller-manager-bootstrap"
+	echo "Rendering Cloud Controller Manager core manifests..."
+
+	rm --recursive --force cloud-controller-manager-bootstrap
+
+	# Copy the CCCMO images configMap to resolve pod images from internal registry in CCCMO render run
+	# Source config map is located in https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/master/manifests/0000_26_cloud-controller-manager-operator_01_images.configmap.yaml
+	bootkube_podman_run \
+	--name copy-cloud-controller-manager-images \
+	--rm -it \
```

Comment:
You probably don't need an interactive terminal on the bootstrap machine, right?  And since in the happy case we will shortly remove the whole machine, I think we want to keep the metadata for debug logging.  Can we drop this line?